src/tab/tabs.ts

```diff
@@ -37,28 +37,46 @@ export class RemixTabs extends LitElement {
   @property({ type: String })
   public defaultIcon: string = '';
 
+  // enables/disables + button to add new tab
   @property({ type: Boolean })
   public canAdd = true;
 
-  private defaultTab() {
+  private generateId(): string {
```

Comment:
I think the generation of Tab should be outside of the component. Like that we keep the component View only, and it's up to the host application to create the tab.